sklearn/feature_selection/rfe.py

```diff
@@ -146,6 +149,13 @@ def fit(self, X, y):
 
         support_ = np.ones(n_features, dtype=np.bool)
         ranking_ = np.ones(n_features, dtype=np.int)
+
+        if step_score:
+            # Determine the number of subsets of features
+            self.scores_ = np.zeros(
```

Comment:
sorry, I don't understand this point. It seems you could save these six lines here...